### ISSUE TITLE
Different configurations with distinct templates

### ISSUE THREAD

**Issue by ArchaicDebugger** [RandomEngy/UnitTestBoilerplateGenerator#1]
### Installed product versions
- Visual Studio: 2019 Community Edition
- This extension: 2.5.0

### Description
Would love to have different configurations for different tests. For example, there are projects where we have different types of tests, and our test classes inherit from different base test classes, some of which might seed data, etc. Definitely love the idea of keeping different configurations with different setups/templates and selecting one of them when creating the test

![image](https://user-images.githubusercontent.com/13509956/108100440-34e0ac00-7086-11eb-9048-ee7aa0da7950.png)

**Comment by ArchaicDebugger** [comment]
Don't get me wrong, this extension is a great value to me as a developer :D would love to share this with anyone

**Comment by RandomEngy** [comment]
Seems like it would be handy. Is this the feature you said you might create a PR to add?

**Comment by ArchaicDebugger** [comment]
Yes, wanted to ask whether it was a good idea first and was within the scope of the project before spending hours working on it

**Comment by RandomEngy** [comment]
Yes, that sounds good and in-scope to me. I would probably hide the configuration drop down unless the user has set up multiple configurations in the options.

One thing to be aware of:
There's a feature to store templates in a local file to share with the team in source control. It would need to support that mode, as well as the mode where we store templates in the built-in Visual Studio settings storage.

**Comment by ArchaicDebugger** [comment]
Definitely, the versioning of the template is crucial in my current project. I'll use a JSON config that adds the feature but does not disrupt anybody's work in case they update the extension.
